.github/ISSUE_TEMPLATE/bug_template.md.txt

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug Report'
+labels: 'needs-triage,needs-sig,kind/bug'
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Try to isolate the issue to help the community to reproduce it easily and increase chances for a fast fix.
+
+**Assets required**
+[!Important!] Please provide sample assets needed to reproduce the issue, either as an [attachment](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files) or as a link to a public asset in Github.
+
+**Steps to reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '...'
+3. Select attached asset '...'
+4. Scroll down to '...'
+5. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Screenshots/Video**
+If applicable, add screenshots and/or a video to help explain your problem.
+
+**Found in Branch**
+Name of or link to the branch where the issue occurs.
+
+**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
+Please provide the SHA/hash that identifies the latest commit from the o3de/o3de repository if the issue is reproducible in the development or other official branches. 
+You can get the commit ID by running the `git rev-parse HEAD` command on your current branch.
+

```

Comment:
I think a commit ID for the o3de-extras repository is also needed here.
We could also give the user an option to provide a release version of o3de if it was installed from the package (not supported by the ROS 2 Gem yet, but will be soon)

Reply:
Good point. I added corresponding parts for o3de-extras.